assets/scss/components/stencil/applePay/_applePay.scss

```diff
@@ -0,0 +1,54 @@
+$apple-pay-black: black;
+$apple-pay-white: white;
+
+.apple-pay-checkout-button {
```

Comment:
That class name comes from bcapp, nothing I can do
![image](https://cloud.githubusercontent.com/assets/1934727/20287192/f41007be-aa7f-11e6-9ea4-d69a3b9734bf.png)